README.md

```diff
@@ -53,347 +40,9 @@ Articles about dotenv-linter:
-
-```shell script
-$ dotenv-linter --exclude .env.test
-.env:2 DuplicatedKey: The FOO key is duplicated
-.env:3 UnorderedKey: The BAR key should go before the FOO key
-```
-
-If you need to skip some checks, you can use the argument `--skip CHECK_NAME` or its short version `-s CHECK_NAME`:
-
-```shell script
-$ dotenv-linter --skip UnorderedKey EndingBlankLine
-.env:2 DuplicatedKey: The FOO key is duplicated
-```
-
-If you need to view all available checks, you can use the argument `--show-checks`:
-
-```shell script
-$ dotenv-linter --show-checks
-DuplicatedKey
-EndingBlankLine
-ExtraBlankLine
-IncorrectDelimiter
-KeyWithoutValue
-LeadingCharacter
-LowercaseKey
-QuoteCharacter
-SpaceCharacter
-TrailingWhitespace
-UnorderedKey
-```
-
-## ✅ Checks
-
-### Duplicated Key
-
-Detects if a key is not unique:
-
-```env
-❌ Wrong
-FOO=BAR
-FOO=BAR
-
-✅ Correct
-FOO=BAR
-BAR=FOO
-```
-
-### Ending Blank Line
-
-Detects if a file doesn't have a blank line at the end:
-
-```env
-❌ Wrong
-FOO=BAR
-```
-
-```env
-✅ Correct
-FOO=BAR
-
-```
-
-### Extra Blank Line
-
-Detects if a file contains more than one blank line in a row:
-
-```env
-❌ Wrong
-A=B
-
-
-FOO=BAR
-```
-
-```env
-❌ Wrong
-A=B
-FOO=BAR
-
-
-```
-
-```env
-✅ Correct
-A=B
-
-FOO=BAR
-
-```
-
-```env
-✅ Correct
-A=B
-FOO=BAR
-
-```
-
-### Incorrect delimiter
-
-Detects if a key does not use an underscore to separate words:
-```env
-❌ Wrong
-FOO-BAR=FOOBAR
-
-✅ Correct
-FOO_BAR=FOOBAR
-```
-
-### Key without value
-
-Detects if a line has a key without a value:
-```env
-❌ Wrong
-FOO
-
-✅ Correct
-FOO=
-
-✅ Correct
-FOO=BAR
-```
-
-### Leading character
-
-Detects if a line starts with an unallowed character (characters from `A` to `Z` and `_` (underscore) are allowed):
-
-```env
-❌ Wrong
- FOO=BAR
-
-❌ Wrong
-.FOO=BAR
-
-❌ Wrong
-*FOO=BAR
-
-❌ Wrong
-1FOO=BAR
-
-✅ Correct
-FOO=BAR
-
-✅ Correct
-_FOO=BAR
-```
-
-### Lowercase key
-
-Detects if a key has lowercase characters:
-
-```env
-❌ Wrong
-FOo_BAR=FOOBAR
-
-❌ Wrong
-foo_bar=FOOBAR
-
-✅ Correct
-FOO_BAR=FOOBAR
-```
-
-### Quote character
-
-Detects if a value is wrapped in quotes:
-
-```env
-❌ Wrong
-FOO="BAR"
-
-❌ Wrong
-FOO='BAR'
-
-✅ Correct
-FOO=BAR
-```
-
-### Space character
-
-Detects lines with a whitespace around equal sign character `=`:
-
-```env
-❌ Wrong
-FOO =BAR
-
-❌ Wrong
-FOO= BAR
-
-❌ Wrong
-FOO = BAR
-
-✅ Correct
-FOO=BAR
-```
-
-### Trailing whitespace
-
-Detects if a line has a trailing whitespace.
-
-### Unordered Key
-
-Detects if a key is not alphabetically ordered:
-
-```env
-❌ Wrong
-FOO=BAR
-BAR=FOO
-
-✅ Correct
-BAR=FOO
-FOO=BAR
-```
-
-You can use blank lines to split lines into groups (will be available in [v2.1.0](https://github.com/dotenv-linter/dotenv-linter/issues/217)):
-
-```env
-❌ Wrong
-FOO=BAR
-BAR=FOO
-
-✅ Correct 
-FOO=BAR
+## 📄 Documentation
 
-BAR=FOO
-```
+Documentation is live hosted at https://dotenv-linter.github.io
```

Comment:
It seems to me that placing a table of contents documentation with links would be a good solution. 
The table of contents changes much less frequently, but will give an understanding of the presence of sections of interest.
What do you think?